x-pack/test/functional/services/cloud_security_posture/benchmarks.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import type { FtrProviderContext } from '../../ftr_provider_context';
+
+export function BenchmarksProvider({ getService, getPageObjects }: FtrProviderContext) {
+  const testSubjects = getService('testSubjects');
+  const retry = getService('retry');
+  const PageObjects = getPageObjects(['common']);
+
+  /** Actions */
+  const filterBenchmarks = (value: string) =>
+    testSubjects.setValue('benchmark-search-field', value);
+
+  const setPageSize = async (size: 10 | 25) => {
+    await testSubjects.click('tablePaginationPopoverButton');
+    await testSubjects.click(`tablePagination-${size}-rows`);
+  };
+
+  const toggleSortByName = async () => {
+    // TODO: use custom test subject
+    await testSubjects.click('tableHeaderCell_package_policy.name_0');
+  };
+
+  /** Navigation */
+  const goToBenchmarkPage = () =>
+    PageObjects.common.navigateToUrl('securitySolution', 'cloud_security_posture/benchmarks', {
+      shouldUseHashForSubUrl: false,
+    });
+
+  const goToNextPage = async () => {
+    await testSubjects.click('pagination-button-next');
+    await testSubjects.isEnabled('pagination-button-0');
+  };
+
+  const goToPreviousPage = async () => {
+    await testSubjects.click('pagination-button-previous');
+    await testSubjects.isEnabled('pagination-button-1');
+  };
```

Comment:
we probably shouldn't be referencing test-subjects from eui but it seems common 

we might be able to use our own ids